plugins/automox/help.md

```diff
@@ -21,7 +21,7 @@ outcomes on endpoints, and basic Automox platform administration.
 
 # Supported Product Versions
 
-* All as of 12/17/2021
+* All as of 1/21/2021
```

Comment:
Confirming that this indicates that supported versions has been expanded to include versions as far back as January of last year?